lldb/packages/Python/lldbsuite/test/gdbclientutils.py

```diff
@@ -306,7 +309,8 @@ def haltReason(self):
         # SIGINT is 2, return type is 2 digit hex string
         return "S02"
 
-    def qXferRead(self, obj, annex, offset, length):
+    def qXferRead(self, obj: str, annex: str, offset: int,
+                  length: int) -> tuple[str | None, bool]:
```

Comment:
In future please separate annotations of existing types into their own PR. If you're changing the type in the process of doing something then it's fine to include in the same PR.